### PR TITLE
Customize OpenAPI component styling

### PR DESCRIPTION
**Pull request by aerfio**
**Description**

Changes proposed in this pull request:

- make OpenAPI component styling consistent with rest of the ConsoleUI

**Related issue(s)**

See also #684

### CODE REVIEW

**Comment by aerfio** [comment]
To test this use instruction from [here](https://github.com/kyma-project/bundles/releases/tag/0.6.0).

**Comment by aerfio** [comment]
Those custom swagger components are copy pasted from https://github.com/swagger-api/swagger-ui with only slight needed changes btw

File: catalog/src/components/ServiceClassDetails/SwaggerApi/plugins.js [hunk 1]
```diff
@@ -1,33 +1,41 @@
...
-      },
+      return <Original {...customProps} />;
+    },
+    authorizeBtn: (Original, system) => props => {
+      return null;
+    },
+    authorizeOperationBtn: (Original, system) => props => {
+      return null;
+    },
+    info: (Original, system) => props => {
```
Comment (magicmatatjahu) [inline catalog/src/components/ServiceClassDetails/SwaggerApi/plugins.js]:
```js
props => null
```

File: catalog/src/components/ServiceClassDetails/SwaggerApi/plugins.js [hunk 2]
```diff
@@ -1,33 +1,41 @@
...
-      },
-      info: (Original, system) => props => {
-        return <div />;
-      },
+      return <Original {...customProps} />;
+    },
+    authorizeBtn: (Original, system) => props => {
+      return null;
+    },
+    authorizeOperationBtn: (Original, system) => props => {
```
Comment (magicmatatjahu) [inline catalog/src/components/ServiceClassDetails/SwaggerApi/plugins.js]:
```js
props => null
```

File: catalog/src/components/ServiceClassDetails/SwaggerApi/plugins.js [hunk 3]
```diff
@@ -1,33 +1,41 @@
...
-      },
-      authorizeBtn: (Original, system) => props => {
-        return <div />;
-      },
-      info: (Original, system) => props => {
-        return <div />;
-      },
+      return <Original {...customProps} />;
+    },
+    authorizeBtn: (Original, system) => props => {
```
Comment (magicmatatjahu) [inline catalog/src/components/ServiceClassDetails/SwaggerApi/plugins.js]:
```js
props => null
```

File: catalog/src/components/ServiceClassDetails/SwaggerApi/plugins.js [hunk 4]
```diff
@@ -1,33 +1,41 @@
...
+      return null;
+    },
+    authorizeOperationBtn: (Original, system) => props => {
+      return null;
+    },
+    info: (Original, system) => props => {
+      return null;
     },
+    Col: SchemesWrapper,
+    Models: Models,
```
Comment (magicmatatjahu) [inline catalog/src/components/ServiceClassDetails/SwaggerApi/plugins.js]:
```js
Models
```

File: instances/src/components/ServiceInstanceDetails/SwaggerApi/SwaggerCustomComponents/SchemesWrapper.jsx [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+  letter-spacing: normal;
+  color: #32363a;
+`;
+
+export const SchemesWrapper = (Orig, system) => props => {
+  if (props.className === 'schemes wrapper') {
+    return (
+      <Wrapper>
+        <Text>API Console</Text>
+        <ContentWrapper className="test">
```
[the remark this answers is not in the document]
Comment (aerfio) [reply instances/src/components/ServiceInstanceDetails/SwaggerApi/SwaggerCustomComponents/SchemesWrapper.jsx]: Good catch

**Pull request merged**